pipeline_dp/combiners.py

```diff
@@ -392,6 +399,80 @@ def explain_computation(self) -> ExplainComputationReport:
         return lambda: f"Computed variance with (eps={self._params.eps} delta={self._params.delta})"
 
 
+class QuantileCombiner(Combiner):
+    """Combiner for computing DP quantiles.
+
+    It can compute any number of quantiles at once with the same accuracy.
+    It uses QuantileTree algorithm from Google C++ DP library
+    https://github.com/google/differential-privacy/blob/main/cc/algorithms/quantile-tree.h
+    with the wrapper from PyDP.
+
+    The accumulator is QuantileTree object serialized to string.
+    """
+
+    AccumulatorType = str
+
+    def __init__(self, params, percentiles: List[float]):
```

Comment:
Can we call this "percentiles_to_compute" to differentiate from actual percentiles? Also for quantiles, and in other places.

It's a bit longer but less confusing, especially that you use "quantiles" to mean something else right below (line 438).